GitHubTokenSource.cs

```diff
@@ -21,7 +21,7 @@ public static string GetAccessToken()
 
             if (string.IsNullOrEmpty(token))
             {
-                throw new Exception("GitHub access token is missing; please put it in tools/.githubtoken, or in the GITHUB_TOKEN environment variable.");
+                throw new Exception("GitHub access token is missing; please put it in <shared tools directory>/.githubtoken, or in the GITHUB_TOKEN environment variable.");
```

Comment:
I used a placeholder instead of `shared-tools`, because I don't want to make assumptions on where the submodule lives in the parent repo

Reply:
This makes sense. However, we do calculate where we expect the file to be. Can we not just output that value?

Reply:
Oh! Yes, of course.